src/pages/Lend/LendByCategoryPage.vue

```diff
@@ -546,15 +588,18 @@ export default {
 				} else {
 					try {
 						return this.apollo.query({
-							query: loanChannelQuery,
+							query: this.categoryServiceExpActive ? categoryServiceRowsQuery : loanChannelQuery,
 							variables: {
 								ids: [category.id],
 								imgDefaultSize: this.showHoverLoanCards ? 'w480h300' : 'w480h360',
 								imgRetinaSize: this.showHoverLoanCards ? 'w960h600' : 'w960h720',
 							},
 						}).then(({ data }) => {
-							const fetchedCategory = data?.lend?.loanChannelsById?.[0];
-							if (fetchedCategory?.loans?.values?.length) {
+							const fetchedCategory = this.categoryServiceExpActive
+								? data?.loanCategoriesByLoanChannelIds?.[0]
+								: data?.lend?.loanChannelsById?.[0];
+							if (fetchedCategory?.loans?.values?.length
+								|| fetchedCategory?.savedSearch?.loans?.values?.length) {
```

Comment:
Checking our additional savedSearch data structure for loans brought back the lazy loading funcitonality! Previously we were fetching the categories but not rendering them b/c they weren't added to the `realCategories` array.